conversions/time_conversions.py

```diff
@@ -0,0 +1,57 @@
+"""
+A unit of time is any particular time interval, used as a standard
+way of measuring or expressing duration.
+The base unit of time in the International System of Units (SI),
+and by extension most of the Western world, is the second,
+defined as about 9 billion oscillations of the caesium atom.
+
+WIKI: https://en.wikipedia.org/wiki/Unit_of_time
```

Comment:
Let's use our 88 characters per line...
```suggestion
A unit of time is any particular time interval, used as a standard way of measuring or
expressing duration.  The base unit of time in the International System of Units (SI),
and by extension most of the Western world, is the second, defined as about 9 billion
oscillations of the caesium atom.

https://en.wikipedia.org/wiki/Unit_of_time
```